dll/cpl/input/input.h

```diff
@@ -94,4 +98,6 @@ DWORDfromString(const WCHAR *pszString)
 
 VOID GetSystemLibraryPath(LPWSTR pszPath, INT cchPath, LPCWSTR pszFileName);
 
+extern BOOL g_bRebootNeeded;
```

Comment:
Duplicate of the other one above in the file.

Reply:
Fixed.